### DISCUSSION TITLE
Nonprofit Application and Copilot & Codespace Application and Use?

### DISCUSSION THREAD

**Discussion by oriemaharjfamilyfoundation** [community/events#52] [Cloud]
So I think after Ryan Salva mention Nerds Nerds Nerds, hopefully I have the right guy, anyway, that guy, stated nonprofits can use Copilot but he didn't say how. So right now nonprofits can apply for sponsorship/aid through Github Social Impact, which gives us access to more memory for our nonprofit needs and help when we need it. Would codespace be included? is it already? Would copilot be included or would they have some sort of discounted program for copilot?

**Comment by ossanna16** [comment]
Hi there @oriemaharjfamilyfoundation! You can apply for a free GitHub Enterprise Account via [GitHub Social Impact](https://socialimpact.github.com). Currently Codespaces would be included in that account but not Copilot as of yet as Copilot is only available to individuals at the moment. Per the [Copilot site](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/), it should be available via Enterprise later this year.

**Reply by oriemaharjfamilyfoundation** [reply]
Thank you Anna :)
